include/random_walks/gaussian_accelerated_billiard_walk.hpp

```diff
@@ -67,9 +68,15 @@ struct GaussianAcceleratedBilliardWalk
     struct Walk
     {
         typedef typename Polytope::PointType Point;
-        typedef typename Polytope::DenseMT DenseMT;
+        typedef typename Polytope::MT MT;
+        typedef typename Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic> DenseMT;
         typedef typename Polytope::VT VT;
         typedef typename Point::FT NT;
+        using AA_type = std::conditional_t< std::is_same_v<MT, typename Eigen::SparseMatrix<NT, Eigen::RowMajor>>, typename Eigen::SparseMatrix<NT>, DenseMT >; 
```

Comment:
similar comment to `AE_type`, is there a better naming?

Reply:
hmm, I'll think of one, I'm not really sure what name I could give it but I'll see if I can come up with a better name.